packages/@uppy/webcam/src/style.scss

```
@@ -90,6 +90,12 @@
   margin-right: 12px;
 }
 
+.uppy-Webcam-recordingLength {
+  position: absolute;
+  right: 20px;
+  color: $gray-600;
```

```suggestion
color: $gray-600;
font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
```

How about using a mono-spaced font, so the numbers don’t jump when they change? They jump slightly for me in Firefox on MacOS.

---

Agreed, thanks. I have applied your suggested commit.